templates/flamegraph.mustache

```diff
@@ -73,16 +75,22 @@ function init() {
         draw();
     }
 
-    window.excimerChart.label(function(d) {
-        return d.data.name + ' (' + ((d.x1 - d.x0) * 100).toFixed(3)
-                + '%, ' + d.value + ' samples, duration ~' +  d.value * {{samplerate}} + ' ms)'
-    });
+    displaysamplestats = function (d) {
+        time = d.value * {{samplerate}};
+        seconds = time / 1000;
+        minutes = Math.floor(seconds / 60);
+        hours = Math.floor(minutes / 60);
+        minutes = minutes % 60;
+        seconds = (seconds % 60).toFixed(3);
+        durationstr = (hours ? (hours + 'h ') : '') + (minutes ? (minutes + 'm ') : '') + seconds + 's';
```

Comment:
This line is a bit hard to read. Are we able to split this out and clean it up a bit. This also applies to the return statement further below. There's a bunch of calculation happening within the same line of code.
```suggestion
        let durationstr = '';
        if (hours) {
            durationstr += `${hours}h `;
        }
        if (minutes) {
            durationstr += `${minutes}m `;
        }
        if (seconds) {
            durationstr += `${seconds}s`;
        }
```

Reply:
done